transformer_engine/pytorch/attention/dot_product_attention/utils.py

```diff
@@ -1897,22 +1908,139 @@ def get_attention_quantizers(fp8, quantizers, cp_specific_quantizers=False):
     dP_quantizer = quantizers["scaling_bwd"][META_DP]
     dP_quantizer.set_usage(rowwise=True, columnwise=False)
     dP_quantizer.interal = True
-    dQKV_CP_quantizer = quantizers["scaling_bwd"][META_DQKV_CP]
-    dQKV_CP_quantizer.set_usage(rowwise=True, columnwise=False)
-    dQKV_CP_quantizer.internal = True
-    O_CP_quantizer = quantizers["scaling_fwd"][META_O_CP]
-    O_CP_quantizer.set_usage(rowwise=True, columnwise=False)
-
-    if cp_specific_quantizers:
-        return (
-            QKV_quantizer,
-            O_quantizer,
-            O_CP_quantizer,
-            S_quantizer,
-            dQKV_quantizer,
-            dQKV_CP_quantizer,
-            dO_quantizer,
-            dP_quantizer,
-        )
 
     return QKV_quantizer, O_quantizer, S_quantizer, dQKV_quantizer, dO_quantizer, dP_quantizer
+
+
+def print_quantizers(
+    label,
+    layer_number,
+    QKV_quantizer,
+    O_quantizer,
+    S_quantizer,
+    dQKV_quantizer,
+    dO_quantizer,
+    dP_quantizer,
+):
+    """Print the type and scale/amax of attention quantizers"""
+    names = [
+        "QKV_quantizer",
+        "S_quantizer",
+        "O_quantizer",
+        "dO_quantizer",
+        "dP_quantizer",
+        "dQKV_quantizer",
+    ]
+    quantizers = [
+        QKV_quantizer,
+        S_quantizer,
+        O_quantizer,
+        dO_quantizer,
+        dP_quantizer,
+        dQKV_quantizer,
+    ]
```

Comment:
Could you move the creation of those lists inside the if statement to not increase the CPU overhead?